src/styles/styles.css

```diff
@@ -1,13 +1,22 @@
-html {
-  overflow: auto;
-  font-family: "Poppins" !important;
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap");
```

Comment:
We should get fonts using fontSource. See https://fontsource.org/fonts/poppins/install for more details

Reply:
Yeah fixed

Reply:
I do not see entry in `package.json` to install the font from `npm`